src/main/java/javax/validation/Validation.java

```diff
@@ -273,7 +275,7 @@ public ValidationProviderResolver getDefaultValidationProviderResolver() {
 			if ( validationProviders.size() == 0 ) {
 				String msg = "Unable to create a Configuration, because no Bean Validation provider could be found." +
```

Comment:
Agreed, it won't change much. If anything, it should be done for all the messages used in this class. @otaviojava, if like, send in a PR for that :)